src/BingChat/BingChatClientOptions.cs

```diff
@@ -32,4 +34,9 @@ public sealed class BingChatClientOptions
     /// Tone used for conversation (Default: Balanced)
     /// </summary>
     public BingChatTone Tone { get; set; }
+    
+    /// <summary>
+    /// Optional Proxy used for requests
+    /// </summary>
+    public WebProxy? Proxy { get; set; } = new WebProxy();
```

Comment:
Property initializer seems to be invalid here because it contradicts the logic in handler creation `UseProxy = _options.Proxy != null`.

Reply:
Thanks for pointing that out I have fixed it. I think my brain was on autopilot. Tested with the change and works with no proxy given or a http proxy given